docs/source/conf.py

```diff
@@ -70,6 +71,7 @@
     'sphinxcontrib.datatemplates',
     'sphinxcontrib.bibtex',
     'nbsphinx',
+    'myst_nb',
```

Comment:
please add in alphabetical order

Reply:
adjusted in [`d634a56` (#1337)](https://github.com/pymovements/pymovements/pull/1337/commits/d634a5668a247deda7200cc6c3f0b8dcaa7b2eaf)